docs/src/app/components/pages/components/Stepper/HorizontalLinearStepper.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import {
+  Stepper,
+  HorizontalStep as Step,
+} from 'material-ui/lib/stepper';
+
+import {
+  Paper,
+  FontIcon,
+  RaisedButton,
+  FlatButton,
+} from 'material-ui';
```

Comment:
Could we require the component individually? That will reduce the build size for people using this example.